pkspace/utils/loaders.py

```diff
@@ -0,0 +1,105 @@
+import glob
+import json
+import os
+import xml.etree.ElementTree
+import cv2
+import numpy as np
+
+
+class Loader:
+    def load(self, path, split):
+        pass
+
+
+class PKspaceLoader(Loader):
+    def __get_spaces__(self, src, desc):
```

Comment:
@rstevanak why does this function has so many underscores in it? As far as I see, this makes it  a [Special Name](https://docs.python.org/3.4/reference/datamodel.html#specialnames), which is intended for Python's internal use. If we are afraid that this function may be overridden by something that subclasses `PKspaceLoader`, it is probably better to call it `_get_spaces`.

See [this](https://shahriar.svbtle.com/underscores-in-python) for more details.